metannotate-wrapper-docker

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# Created March 10, 2017, by Jackson Tsuji (Neufeld lab PhD student)
+# Description: Simple wrapper for running MetAnnotate command line in Docker container.
+# Even simpler than metannotate-wrapper.
+
+# Basic script stuff (Buffalo, 2015):
+set -e
+set -u
+set -o pipefail
```

Comment:
These sets could be combined onto one command call. Place directly below the shebang.

https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/